src/main/fc/init.c

```diff
@@ -999,5 +999,9 @@ void init(void)
 
     tasksInit();
 
+#ifdef SIMULATOR_MULTITHREAD
+    rescheduleTask(TASK_SERIAL, 1);
```

Comment:
Isn't it possible to use correct value of `serialConfig()->serial_update_rate_hz`? It is used in tasksInit.

Also it may be better to move this code to https://github.com/betaflight/betaflight/blob/c16e971825e92f2a742f1bf669b22f65e68cfa45/src/main/fc/tasks.c#L443

Reply:
Well, I found remove these lines also works.